docs/public/google67c4a117546ddf5e.html

```diff
@@ -0,0 +1 @@
+google-site-verification: google67c4a117546ddf5e.html
```

Comment:
This is for google search console verification where we can submit our sitemap. This should be able to deleted once we we are verified.